app/contracts/settings/working_days_params_contract.rb

```
@@ -41,8 +41,11 @@ def working_days_are_present
       end
     end
 
+    # TODO: consider implementing using GoodJob concurrency control mechanisms
     def unique_job
-      if WorkPackages::ApplyWorkingDaysChangeJob.scheduled?
+      if GoodJob::Job
```

I would suggest

1) putting this into a method like `job_scheduled?'
2) using `job_class: WorkPackages::ApplyWorkingDaysChangeJob.name`

The latter makes sure we don't miss this as easily if the job was ever to be renamed or removed because this will then lead to a runtime error rather than the query just silently returning nothing always.